docs/strategy-advanced.md

```diff
@@ -366,7 +393,7 @@ However, freqtrade also offers a custom callback for both order types, which all
 !!! Note
     Unfilled order timeouts are not relevant during backtesting or hyperopt, and are only relevant during real (live) trading. Therefore these methods are only called in these circumstances.
 
-### Custom order timeout example
```

Comment:
I'd say this was pretty much intended - as it indents this in the visual (and outline) hierarchy below `#custom order timeout rules`.